doc/quickstart.md

```diff
@@ -66,6 +66,7 @@ On CentOS and openSuSE:
     yum copr -y enable jhli/ipmctl
     yum copr -y enable jhli/safeclib
     yum install -y libipmctl-devel
+    yum install -y python-devel
```

Comment:
Rather than being a separate `yum install` command, you can probably just add `python-devel` to one of the `yum install` commands above.

Reply:
Done.

Reply:
Sorry, I was not more clear.  It would be appropriate to add it to one of the `yum install` commands above line 64.